misc/factorial_recursive.c

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file factorial_recursive.c
```

Comment:
If we don't add the filename, it'll be better, otherwise we'll need to add the full directory.
```suggestion
 * @file
```